contrib/test/integration/build/cri-o.yml

```diff
@@ -64,7 +64,9 @@
 - name: add docker.io default registry
   lineinfile:
     dest: /etc/crio/crio.conf
-    line: '"docker.io"'
+    line: |
+          # Added by Ansible from build/cri-o.yml
+          'registries = [ "docker.io" ]'
```

Comment:
I don't think you want the single-quotes here, they will be written into the file.